components/about-page-dropdown.js

```diff
@@ -0,0 +1,81 @@
+import { Disclosure } from '@headlessui/react';
+import { ChevronUpIcon } from '@heroicons/react/outline';
+
+export default function AboutPageDropdown({ currentSlug, slugs }) {
+  const remainingTitles = slugs?.filter((el) => el.slug !== currentSlug) ?? [];
+  const currentPage = slugs?.filter((el) => el.slug === currentSlug) ?? [];
+
+  return (
+    <div className='pt-8 w-full'>
+      <Disclosure>
+        {({ open }) => (
+          <>
+            <Disclosure.Button
+              className={`${
+                open ? 'rounded-t-md' : 'rounded-md'
+              } text-black bg-green-500 hover:text-green-500 flex justify-between items-center border border-green-500 h-10 w-full focus:outline-none focus-visible:ring focus-visible:ring-green-500 focus-visible:ring-opacity-75`}
+            >
+              <div
+                className={`${
+                  open ? 'rounded-tl-md' : 'rounded-l-md'
+                } h-full bg-white w-11/12 flex items-center justify-start pl-3 truncate`}
+              >
+                <span className='inline text-left '>{currentPage[0]?.title}</span>
+              </div>
+              <div className='w-1/12 px-5 h-full pr-5 bg-green-500 grid items-center justify-center rounded-r-md border border-green-500'>
+                <ChevronUpIcon
+                  className={`${open ? '' : 'rotate-180 transform'} h-5 w-5 text-white z-10`}
+                />
+              </div>
+            </Disclosure.Button>
+
+            <Disclosure.Panel>
+              <Disclosure.Button as='a' href={`/about/${remainingTitles[0]?.slug}`}>
+                <div className='bg-white text-black-white-800 border-b border-l border-r border-green-500 h-10 flex items-center hover:text-green-500'>
+                  <span className='block pl-3 truncate'>{remainingTitles[0]?.title}</span>
+                </div>
+              </Disclosure.Button>
+              <Disclosure.Button as='a' href={`/about/${remainingTitles[1]?.slug}`}>
+                <div className='bg-white text-black-white-800 border-b border-l border-r border-green-500 h-10 flex items-center hover:text-green-500'>
+                  <span className='block pl-3 truncate'>{remainingTitles[1]?.title}</span>
+                </div>
+              </Disclosure.Button>
+
+              <Disclosure.Button as='a' href={`/about/${remainingTitles[2]?.slug}`}>
+                <div
+                  className={`${
+                    remainingTitles[2] === remainingTitles[-1] ? '' : 'rounded-b-md'
```

Comment:
hey am bit confused why we have `-1` index here?

Reply:
Just to add rounded corners on the last of the about pages. As in if index 2 is also -1 then it is the last.

Reply:
ohhhh